### PR TITLE
Add chart labels to Prometheus.

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Fixes https://github.com/netdata/netdata/issues/15089

This PR is adding chart  labels missed in Prometheus metric.
##### Test Plan

1. Compile Branch
2. Take a look in https://localhost:19999/api/v1/allmetrics?format=prometheus and search for chart labels like the one shown in documentation update.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? Prometheus
- Can they see the change or is it an under the hood? If they can see it, where? Only when netdata ins exposing data for prometheus.
- How is the user impacted by the change? We are sharing more data with external sources.
- What are there any benefits of the change? We can compare netdata with other tools
</details>

### CODE REVIEW

File: exporting/README.md [hunk 1]
```diff
@@ -284,7 +284,7 @@ Configure individual connectors and override any global settings with the follow
     and names are human friendly labels (also unique). Most charts and metrics have the same ID and name, but in several
     cases they are different: disks with device-mapper, interrupts, QoS classes, statsd synthetic charts, etc.
 
--   `send configured labels = yes | no` controls if labels defined in the `[host labels]` section in `netdata.conf`
+-   `send configured labels = yes | no` controls if host labels defined in the `[host labels]` section in `netdata.conf`
```
Comment (ilyam8) [inline exporting/README.md:287]: unnecessary change

> if host labels defined in the `[host labels]`

It is `[host labels]` section, ofc it is host labels

**Review by ilyam8 — APPROVED** [review]

**Review by Ancairon — APPROVED** [review]
